internal/pkg/build/files/copy.go

```diff
@@ -17,21 +17,40 @@ import (
 // checks dst and creates parent directories if they do not exist
 // before calling cp
 func Copy(src, dst string) error {
-	_, err := os.Stat(dst)
+	// resolve any bash globbing in filepath
+	paths, err := expandPath(src)
+	if err != nil {
+		return fmt.Errorf("while expanding source path with bash: %s: %s", src, err)
+	}
+
+	_, err = os.Stat(dst)
 	if os.IsNotExist(err) {
 		// if destination doesn't exist, create parent directories
 		if err := os.MkdirAll(filepath.Dir(dst), 0755); err != nil {
-			return fmt.Errorf("while creating parent directories: %v", err)
+			return fmt.Errorf("while creating parent directories: %s", err)
+		}
+		// if source expands to multiple elements, create destination as directory if it doesnt exist
+		if len(paths) > 1 {
+			if err := os.MkdirAll(dst, 0755); err != nil {
```

Comment:
if we hit this code, it should be creating exactly one directory, dst, as all the parent directories are already there because of line L29.

I would prefer to use os.Mkdir here.